### ISSUE TITLE
Add question: "A disabled element (input, button, etc) fails contrast ration - is this a fail?"

### ISSUE THREAD

**Issue by MaximPerry** [cenw-wscoe/WCAG-in-Plain-Language#12]
Or: "Do disabled elements (input, button, etc) need to pass minimum contrast ratio?"
